src/index.js

```diff
@@ -1,7 +1,8 @@
 const app = require("./app");
 
 const port = process.env.PORT || 5000;
-app.listen(port, () => {
+
+const server = app.listen(port, () => {
```

Comment:
Why this change?

Reply:
This was an attempt to fix tests. I didn't rollback it correctly.